src/i18n/strings/lt.json

```diff
@@ -109,7 +109,7 @@
     "Failed to set Direct Message status of room": "Nepavyko nustatyti tiesioginio pranešimo kambario būklės",
     "Monday": "Pirmadienis",
     "All messages (noisy)": "Visos žinutės (triukšmingas)",
-    "Enable them now": "Įjungti juos dabar",
+    "Enable them now": "Įgalinti juos dabar",
```

Comment:
also a bit accidental? I think these kinds of language fixes should be picked up by weblate come release time.